conda/recipes/ucxx/meta.yaml

```
@@ -40,7 +39,12 @@ requirements:
     - ninja
     - sysroot_{{ target_platform }}
   host:
-    - cudatoolkit ={{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - cudatoolkit
+    {% else %}
+    - cuda-cudart-dev
+    {% endif %}
```

Once the CUDA compiler portion is added, think we can drop the CUDA 11 portion

The CUDA 12 portion is still needed since `libcudart` is dynamically linked to

```suggestion
    {% if cuda_major != "11" %}
    - cuda-cudart-dev
    {% endif %}
```